contrib/examples/deploy.yaml

```diff
@@ -284,7 +280,9 @@ objects:
           - --contention-profiling
           - "false"
           - -v
-          - "10"
+          - "0"
```

Comment:
Can we leave this at 10 for now until everything is switched over to logrus?

Reply:
Can do.

Reply:
Actually FWIW glog still logs things at 0, I was told recently that "0 is more verbose than info" by deads, which I totally do not understand, among several other things with glog.